src/jabs/project/video_manager.py

```diff
@@ -0,0 +1,127 @@
+import json
+import sys
+from pathlib import Path
+
+from jabs.pose_estimation import get_pose_path, get_frames_from_file, open_pose_file
+from jabs.video_reader import VideoReader
+from jabs.video_reader.utilities import get_frame_count
+from .project_paths import ProjectPaths
+from .settings_manager import SettingsManager
+from .video_labels import VideoLabels
+
+
+class VideoManager:
+    def __init__(
+        self,
+        paths: ProjectPaths,
+        settings_manager: SettingsManager,
+        enable_video_check: bool = True,
+    ):
+        """
+        Initialize the VideoManager with paths.
+
+        :param paths: ProjectPaths object containing project directory paths
+        """
+        self._paths = paths
+        self._settings_manager = settings_manager
+        self._videos = []
+        self._total_project_identities = 0
+
+        self._initialize_videos(enable_video_check)
+
+    def _initialize_videos(self, enable_video_check):
+        """Initialize video-related data and perform checks."""
+        self._videos = self.get_videos(self._paths.project_dir)
+        self._videos.sort()
+
+        # self._validate_pose_files()
```

Comment:
Commented code? Explain or remove?

Reply:
good catch -- I think this is the result of some incomplete refactoring